core/src/main/java/org/frankframework/jdbc/JdbcQuerySenderBase.java

```diff
@@ -152,7 +151,13 @@ public enum QueryType {
 		/** To execute Oracle PL/SQL package */
 		PACKAGE,
 		/** For queries that return no data */
-		OTHER
+		OTHER,
+		/** Deprecated: Use OTHER instead */
+		@Deprecated(since = "8.1") INSERT,
```

Comment:
Please add the `@ConfigurationWarning` tag with text that `OTHER` should be used instead.

Reply:
That's not possible inside an enum.

Reply:
Ok, will do.